example_es_responses/index_info.json

```diff
@@ -1,52 +1,56 @@
 {
-  "g-cloud": {
-    "aliases": {
-      "galias": {}
-    },
-    "mappings": {
-      "services": {
-        "dynamic": "strict",
-        "_meta": {
-          "version": "2015-09-28"
+    "g-cloud-9":{
+        "aliases":{
+            "galias":{}
         },
-        "properties": {}
-      }
-    },
-    "settings": {
-      "index": {
-        "creation_date": "1443534824067",
-        "number_of_shards": "5",
-        "number_of_replicas": "1",
-        "version": {
-          "created": "1070199"
+        "mappings":{
+            "services":{
+                "dynamic":"strict",
+                "_meta":{
```

Comment:
Does this _actually_ come back from ES? I guess I'm a bit surprised it doesn't just respond back with something highly processed, i.e. without the things it doesn't understand.

Reply:
Yes, this is what _actually_ comes back.

<img width="1024" alt="screen shot 2017-10-13 at 14 23 04" src="https://user-images.githubusercontent.com/2920760/31548237-1b1c5e2c-b022-11e7-868a-de20c538ede1.png">